.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  NODE_VERSION: 12
+
+jobs:
+  lint:
+    name: Lint files
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Check out a copy of the repo
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Cache npm cache
+        id: cache-dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ secrets.CACHE_VERSION }}-npm-cache
```

Comment:
I added a secret called `CACHE_VERSION` so that we can clear the cache if something goes wrong. Since the value can't be retrieved, I'd recommend setting the value to a random UUID.

Reply:
Once the original cache has been cleared (happens after some number of days), we could remove the secret and update the workflow.